protocol/cosi.go

```diff
@@ -147,44 +149,38 @@ func (pc *ProtocolCosi) Dispatch() error {
 }
 
 // StartAnnouncement will start a new announcement.
-func (pc *ProtocolCosi) StartAnnouncement() error {
+func (pc *CoSi) StartAnnouncement() error {
 	dbg.Lvl3(pc.Name(), "Message:", pc.Message)
 	out := &Announcement{
-		From:         pc.treeNodeID,
-		Announcement: pc.Cosi.CreateAnnouncement(),
+		From: pc.treeNodeID,
```

Comment:
I don't see this used anywhere, and IIRC, sda does provide a `From`-field anyway. Can we delete it here?